### ISSUE TITLE
Built-in slot types newly implemented in non-English languages give "textual" outputs (related to the ticket #252)

### ISSUE THREAD

**Issue by safsoun** [Picovoice/rhino#272]
Hello, 

I've tested the new implemented feature for non-English language (especially French language) but found that outputted values are provided in text strings and not in numbers strings 😢 contrary to the english implementation. 
For example: pv.TwoDigitInteger gives the string "vingt cinq" instead of the string "25".
This is not convenient and complicates the application implementation. 
Could you please replace the text with numbers as done with English implementation?

Thank you 😊

**Comment by ErisMik** [comment]
Closing this ticket as a duplicate of #271

**Comment by safsoun** [comment]
Hello,

Unfortunately, it was not fixed .. I'm still having plain text.
I'm using mcu platform (STM32).

**Comment by mrrostam** [comment]
@safsoun 

We tested the latest libs on our GitHub, and haven't noticed anything related to the mentioned issue with the built-in slot types; for instance, we got '5' instead of 'cinq'. 

Can you share with us the result you're getting using the new libs?

**Comment by safsoun** [comment]
@mrrostam 
I'm still having plain text: "vingt cinq" instead of "25".
I'm using STM32F769I-DISCO board and the output of pv_picovoice_version() is 2.1.0.

**Comment by mrrostam** [comment]
@safsoun That's weird, To make sure that we are on the same page, could you do me a favour and download and use [this lib file](https://github.com/Picovoice/picovoice/blob/master/sdk/mcu/lib/stm32f769/fr/libpicovoice.a), create a new context model using [Picovoice Console](https://console.picovoice.ai/) and see if you still get the same result?

**Comment by safsoun** [comment]
I tested with new lib and had the same issue..
I tried to retrain the model and tested it with the micro on the console and had the issue: "vingt cinq" instead of "25".
Retrained again the model and the issue disappeared! that's weird ..

**Comment by mrrostam** [comment]
Interesting, so the issue is solved, but can I ask you to send me that broken context model (`.rhn` file)?

**Comment by safsoun** [comment]
rhn sent by email.
This is what I did:
I have already a French context that I've created with V1.x. Waited until this issue was fixed.
I downloaded the latest Picovoice lib version 2.1, retrained the model without doing a test with micro on the console.
Loaded the trained model to the board and I had plain text.
Yesterday you requested me to try the lib, I did it but still having the issue.
I came back to the console and did the micro test on the console and I got plain text! I was surprised. So I created another test context (new French context) and used pv.TwoDigitInteger, the issue disappeared! So I came back to my original context and retrained the model (as I did previously), the issue disappeared and had "25" instead of "vingt cinq"!
I don't know if the issue comes from the console itself!
